### DISCUSSION TITLE
In-place upgrade from 2.3.120 to 2.4

### DISCUSSION THREAD

**Discussion by H4waiianPunch** [Security-Onion-Solutions/securityonion#13763] [Unsupported Versions]
Hello,

I'm very slowly trying to work on a PoC to use SO for our business' main IDS platform as opposed to snort/snorby.

I know in a previous blog [found here](https://blog.securityonion.net/2023/08/security-onion-24-has-reached-general.html#:~:text=You%20cannot%20do%20an%20in,work%20for%20network%20installations%20currently.) that an in-place upgrade was not possible.

I was wondering if anything had changed and I'm just not able to find the documentation on it. Ideally I'd be looking to just upgrade from 2.3.120 to 2.4 as it would require less time from my counterparts.

Thanks!

**Comment by dougburks** [comment]
https://docs.securityonion.net/en/2.4/appendix.html
